ruby/private/binary/binary.sh.tpl

```diff
@@ -1,13 +1,34 @@
 #!/usr/bin/env bash
 
+{rlocation_function}
+
+realpath() (
+  OURPWD=$PWD
+  cd "$(dirname "$1")"
+  LINK=$(readlink "$(basename "$1")")
+  while [ "$LINK" ]; do
+    cd "$(dirname "$LINK")"
+    LINK=$(readlink "$(basename "$1")")
+  done
+  REALPATH="$PWD/$(basename "$1")"
+  cd "$OURPWD"
+  echo "$REALPATH"
+)
+export RUNFILES_DIR="$(realpath "${RUNFILES_DIR:-$0.runfiles}")"
```

Comment:
@alexeagle I am no expert with using a runfiles library, but is there any way to avoid using realpath here>

Reply:
Yes, you should use the `rlocation` function you get from 
https://github.com/bazelbuild/bazel/blob/master/tools/bash/runfiles/runfiles.bash#L55-L85

Reply:
Yes, I use `rlocation` below, but it doesn't work unless I export `RUNFILES_DIR` and additionally resolve it with `realpath`.